### PR TITLE
grid

### PR DESCRIPTION
**Pull request by liliya-dev**
[DEMO LINK](https://liliya-dev.github.io/layout_catalog/)
[TEST REPORT LINK](https://liliya-dev.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,309 @@
...
+          <li class="navigation__item">
+            <a class="navigation__item--link" href = "#">Фото</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__item--link" href = "#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```
Comment (V2473) [inline src/index.html:53]: no class

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+  .navigation__item {
+    display: block;
+    width: 100%;
+  }
+
+  .logo {
+    display: block;
+    width: 100%;
+  }
```
Comment (V2473) [inline src/styles/main.scss:41]: should be in header file

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+  margin-top: 50px;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(4, $card-width);
+  grid-column-gap: 48px;
+  grid-row-gap: 51px;
+}
+
+@media screen and (max-width: 900px) and (min-width: 300px) {
```
Comment (V2473) [inline src/styles/main.scss:23]: work bad in bigger resolutions

File: src/styles/main.scss [hunk 4]
```diff
@@ -20,23 +20,14 @@ body {
...
+    grid-template-columns: repeat(auto-fit, $card-width);
   }
+}
 
-  .logo {
-    display: block;
-    width: 100%;
+@media screen and (max-width: 800px) {
+  .main {
+    grid-template-columns: repeat(auto-fit, $card-width);
```
Comment (denkondratiev) [inline src/styles/main.scss]: It still doesn't work on big screens, maybe try without auto-fit. As I understand it, they want us to have four columns in large sizes.

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+  margin-top: 50px;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(4, $card-width);
+  grid-column-gap: 48px;
+  grid-row-gap: 51px;
+}
+
+@media screen and (min-width: 1201px) {
```
Comment (V2473) [inline src/styles/main.scss:23]: screen should stay as 4 columns

File: src/styles/navigation.scss [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    width: 100%;
+    height: 4px;
+    background-color: $blue;
+    bottom: 0;
+    right: 0;
+    border-radius: 8px;
+  }
+}
+
+@media screen and (max-width: 900px) and (min-width: 300px) {
```
Comment (V2473) [inline src/styles/navigation.scss:50]: menu is take all the screen in this mode, make it more compact

File: src/styles/header.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  background: $white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  &__logo {
+    width: $logo-size;
+    height: $logo-size;
+  }
+}
+
+@media screen and (max-width: 900px) and (min-width: 300px) {
```
Comment (V2473) [inline src/styles/header.scss:18]: menu is take all the screen in this mode, make it more compact

**Comment by denkondratiev** [comment]
Yes, it's ok. Only one thing, remember the name of the variables, for example, it is better to use the `main-color` than `blue`, what will happen when you change the color in all cards to red?

**Comment by Vladimir-Beliy** [comment]
/* в PR сравни  новую ветку grid c веткой develop. что бы было видно только как ты меняешь flex на grid */

**Pull request closed without merging**